### PR TITLE
Hide empty properties in view mode

### LINKED ISSUE #3363: Hide empty properties in view mode
### Is your feature request related to a problem? Please describe.

Bildschirmfoto vom 2023-05-03 15-43-54

### Describe the solution you'd like

Hide empty properties

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by GretaD**
fixes #3363 
Screenshot from 2023-05-31 16-46-52

### CODE REVIEW

**Comment by GretaD** [comment]
this might look a bit hack-y but i cannot find any other way that works on both, view and editing mode. Daniel checked it out too and this is the best solution that we could think of.

File: src/components/ContactDetails.vue [hunk 1]
```diff
@@ -569,6 +570,7 @@ export default {
 
 			return ''
 		},
+
```
Comment (kesselb) [inline src/components/ContactDetails.vue:573]:
```suggestion
```

**Review by ChristophWurst — COMMENTED** [review]
https://github.com/nextcloud/contacts/pull/3422/files?w=1 looks good

**Review by ChristophWurst — APPROVED** [review]

**Review by JohannesGGE — APPROVED** [review]

**Comment by st3iny** [comment]
Unit test failures are caused by image pull rate limiting.

**Pull request merged**